File: NotesAppJv/app/src/main/java/com/henriquesbraga/notesappjv/model/Note.java
package com.henriquesbraga.notesappjv.model;

public class Note {

    Integer id;
    String title;
    String note;
    Integer color;
    String date;
    Boolean success;
    String message;

    public Note(Integer id, String title, String note, Integer color){
        this.id = id;
        this.title = title;
        this.note = note;
        this.color = color;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    public Integer getColor() {
        return color;
    }

    public void setColor(Integer color) {
        this.color = color;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: NotesAppKt/app/src/main/java/com/henriquesbraga/notesappkt/model/Note.kt
package com.henriquesbraga.notesappkt.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Note(
    id: Int?,
    title: String,
    note: String,
    color: Int
) {

    @Expose
    @SerializedName("id")
    var id = id

    @Expose
    @SerializedName("title")
    var title = title

    @Expose
    @SerializedName("note")
    var note = note

    @Expose
    @SerializedName("color")
    var color = color

    @Expose
    @SerializedName("date")
    var date = ""

    @Expose
    @SerializedName("success")
    var success = false

    @Expose
    @SerializedName("message")
    var message = ""


}